src/TinyMqtt.cpp

```diff
@@ -533,6 +541,22 @@ if (mesg->type() != MqttMessage::Type::PingReq && mesg->type() != MqttMessage::T
 				debug("end loop");
 				bclose = false;
 				// TODO SUBACK
+				if ((mesg->type() & 0XF0) == MqttMessage::Type::Subscribe)
```

Comment:
It is possible to remove the if
sMQTTMessage msg((mesg->type() & 0XF0) == MqttMessage::Type::Subscribe ? SubAck : UnSuback);
msg.add(header[0]);
msg.add(header[1]);
msg.add(qos);
msg.sendTo(this);